Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj

```diff
@@ -97,7 +97,7 @@
     <Compile Include="Tiled\TmxTerrain.cs" />
     <Compile Include="Tiled\TmxTile.cs" />
     <Compile Include="Tiled\TmxTileOffset.cs" />
-    <Compile Include="Tiled\TmxTileset.cs" />
+    <Compile Include="Tiled\TmxTileSet.cs" />
```

Comment:
You did something silly....

You merged this and changed the filename so now you get the exact same problem, but now you reversed where the big letter occurs....